pokedex.html

```diff
@@ -0,0 +1,16 @@
+<html>
+  <head>
+    <title>Gen 3 Pokedex</title>
+    <link href="pokedex.css" rel="stylesheet" type="text/css" />
```

Comment:
Looks like your missing two meta tags, the first is for the charset, and the second is for the viewport.